eng/Versions.props

```diff
@@ -37,6 +37,8 @@
   <!-- Toolset Dependencies -->
   <PropertyGroup>
     <DotNetCliVersion>3.1.100</DotNetCliVersion>
+    <MicrosoftNetCompilersToolsetVersion>3.3.1-beta3-final</MicrosoftNetCompilersToolsetVersion>
```

Comment:
This seems to be between what it was and what we were talking about changing it to. Did someone update the version of Roslyn for the tests and skip the build?

Reply:
Isn't this what it was in the arcade we're using?

Reply:
I unfortunately don't remember what I was thinking when I wrote that, but 3.3.1-beta3-final looks right to me right now. <shrug>